server/config/config.go

```diff
@@ -70,16 +72,23 @@ func LoadEnvironment() {
 	}
 }
 
-func LoadConfig(name string, config interface{}) {
-	LoadEnvironment()
-
-	log.Info().Msgf("TIGRIS_ENVIRONMENT=%s", GetEnvironment())
-
-	viper.SetConfigName(name)
-	viper.SetConfigType("yaml")
+func LoadConfig(config interface{}) {
+	pflag.StringVarP(&configFile, "config", "c", "", "server configuration file")
+	if err := viper.BindPFlags(pflag.CommandLine); err != nil {
+		log.Err(err).Msg("bind flags")
```

Comment:
do we ignore error here?
if so we can just put `ulog.E(viper.BindPFlags(pflag.CommandLine))` instead of it.

Reply:
Updated! Thanks for the advice!